pkg/syncer/metadatasyncer.go

```diff
@@ -508,6 +532,51 @@ func ReloadConfiguration(metadataSyncer *metadataSyncInformer) error {
 	return nil
 }
 
+// ReconnectVC re-establishes VC connection
+func ReconnectVC(metadataSyncer *metadataSyncInformer) error {
```

Comment:
Same question here about delta with `ReloadConfiguration`

Reply:
I refactored the current ReloadConfiguration function to be re-used for ca file rotation